Makefile

```diff
@@ -213,17 +236,58 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
+update-crd:
+	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > ./deploy/operator/00_scylla.scylladb.com_scyllaclusters.yaml
+.PHONY: update-crd
+
+verify-crd:
+	$(call check-target-applied,update-crd)
+.PHONY: verify-crd
+
+update-dev-deploy:
+	mkdir -p deploy/manager/dev
+	rm -rf deploy/manager/dev/*.yaml
```

Comment:
below there's a `cp` which in case of filename change would not remove old filename. I could use rsync, but i don't want yet another binary dependency.

Reply:
I see that that wouldn't happen automatically, but verify would still catch it so we'd be good. In theory not all the yaml files have to be generated, like we could have distinct config maps or so in the future.

in case you still want to prune, make uses `$(RM) -r`